plugins/main/server/routes/wazuh-elastic.test.ts

```diff
@@ -3,10 +3,19 @@
 import axios from 'axios';
 import { PLUGIN_PLATFORM_REQUEST_HEADERS } from '../../common/constants';
 
-function buildAxiosOptions(method: string, path: string, data: any = {}, headers: any = {}) {
+function buildAxiosOptions(
+  method: string,
+  path: string,
+  data: any = {},
+  headers: any = {},
```

Comment:
We can use `Record<string, unknown>` instead

Reply:
Will not change this as it is out of the scope of this PR